packages/api/portfolioDrafts/portfolio/createPortfolioStep.test.ts

```diff
@@ -30,6 +25,7 @@ const mockResponse = {
 };
 
 describe("Dynamodb mock validation", function () {
+  /*
```

Comment:
Let's not comment out large blocks of code, especially without explanation. Can we just delete this?